lib/service/certreloader_test.go

```diff
@@ -0,0 +1,231 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package service
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509/pkix"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/gravitational/teleport/lib/utils"
+)
+
+func TestCertReloader(t *testing.T) {
```

Comment:
It seems safe to add `t.Parallel()` to this test and given we generate certs, maybe this is worth doing.

Reply:
Done in 6a030b8f04232190222e51415bafe8d5f1236bcc.